guides/common/modules/proc_comparing-content-view-versions.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Comparing_Content_View_Versions_{context}"]
+= Comparing Content View Versions
+
+Use this procedure to compare Content View Version functionality for {Project}.
+
+.Procedure
+* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click Compare.
+
+The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```

Comment:
LGTM! One last suggestion to align the paragraph, and then I believe this is good to go.

```suggestion
+
The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```